core/trino-main/src/test/java/io/trino/operator/aggregation/TestApproximateSetGenericDouble.java

```diff
@@ -33,4 +35,27 @@ protected Object randomValue()
     {
         return ThreadLocalRandom.current().nextDouble();
     }
+
+    @Override
+    protected List<Object> getResultStabilityTestSample()
+    {
+        return ImmutableList.of(
+                Double.MIN_VALUE,
+                -1342434.33434,
+                -15.3,
+                -1.0,
+                17.3,
+                14343437.343434,
+                Double.NaN,
+                Double.MAX_VALUE,
+                Double.NEGATIVE_INFINITY,
+                Double.POSITIVE_INFINITY);
+    }
+
+    @Override
+    protected String getResultStabilityExpected()
+    {
+        // This value should not be changed; it is used to test result stability for $approx_set
+        return "AgwKAAHQJw4Cr3cdAkVnK8A/izJAiXR9wHAwfgHLKZ/Dby+hgbCt6UCdPfo=";
```

Comment:
nit:
i'd consider hex (base16) for more direct representation of the slice produced.